main.go

```diff
@@ -119,6 +123,12 @@ func processExcludePods() (map[string]string, error) {
 }
 
 func processTaint() (*v1.Taint, error) {
+	if len(*annotationVar) != 0 && len(*taintVar) != 0 {
```

Comment:
This works. But I would move both this check and the check for both being empty before processTaint(). Seems clearer that way. But I leave that upto you.

Reply:
Good point. Updated the logic